global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "3.1.100-preview1-014400"
+    "version": "3.1.100"
   },
   "tools": {
-    "dotnet": "3.1.100-preview1-014400",
+    "dotnet": "3.1.100",
```

Comment:
Isn't this updated automatically? Is there a risk that your change here will get overwritten by automation?

Reply:
I'm not sure about this, but it will come from a PR from maestro and we can decide what to do there.